browser/extensions/api/brave_talk_api.cc

```diff
@@ -23,8 +21,7 @@ ExtensionFunction::ResponseAction BraveTalkIsSupportedFunction::Run() {
     return RespondNow(Error("Not available in Tor profile"));
   }
 
-  bool is_supported =
-      base::FeatureList::IsEnabled(brave_talk::features::kBraveTalk);
+  bool is_supported = true;
```

Comment:
nit: how about  directly like `return RespondNow(OneArgument(base::Value(true)));`?
Or, if it's for documentation purpose, constant(`constexpr`) would be better?

Reply:
Good call! I'll mark PR as CI/skip and push a fix